templates/hoover.nomad

```diff
@@ -104,104 +104,42 @@ job "hoover" {
       upstream='hoover-search',
     ) }
 
-  group "collections" {
-    task "nginx" {
+  group "load-balancer" {
```

Comment:
I liked the previous name, `collections`, it was more descriptive. Maybe `collections-lb` if you must?